main.go

```diff
@@ -136,11 +130,6 @@ func main() {
 							EnvVar: "SAM_TEMPLATE_FILE",
 						},
 						cli.StringFlag{
-							Name:   "parameter-values",
```

Comment:
Why remove this parameter?

Reply:
This and the bottom one were mistakes. I'll fix the other one as well